src/intelligence_layer/connectors/document_index/document_index.py

```diff
@@ -501,11 +503,24 @@ def create_index(
 
         data = {
             "chunk_size": index_configuration.chunk_size,
+            "chunk_overlap": index_configuration.chunk_overlap,
             "embedding_type": index_configuration.embedding_type,
         }
         response = requests.put(url, data=dumps(data), headers=self.headers)
         self._raise_for_status(response)
 
+    def delete_index(self, index_path: IndexPath) -> None:
```

Comment:
could technically be a feature in the changelog

Reply:
I added a bullet point